src/main/thrift/parquet.thrift

```diff
@@ -216,13 +216,22 @@ struct Statistics {
    /** count of distinct values occurring */
    4: optional i64 distinct_count;
    /**
-    * Min and max values for the column, determined by its ColumnOrder.
+    * lower and upper bound values for the column, determined by its ColumnOrder.
+    * These may be the actual minimum and maximum values found on a column chunk,
+    * but can also be (more compact) values that do not exist on a column chunk.
+    * For example, instead of storing "Blart Versenwald III", a writer may set
+    * min_value="B", max_value="C". Such more compact values must still be valid
+    * values within the column's logical type.
```

Comment:
```suggestion
    * Lower and upper bound values for the column, determined by its ColumnOrder.
    *
    * These may be the actual minimum and maximum values found on a page or column
    * chunk, but can also be (more compact) values that do not exist on a page or
    * column chunk. For example, instead of storing "Blart Versenwald III", a writer
    * may set min_value="B", max_value="C". Such more compact values must still be
    * valid values within the column's logical type.
```